pyvista/core/_validation/validate.py

```diff
@@ -548,9 +548,11 @@ def validate_rotation(
     check_contains(
         ['right', 'left', None], must_contain=must_have_handedness, name='must_have_handedness'
     )
+
     rotation_matrix = validate_transform3x3(rotation, name=name)
-    if not np.allclose(np.linalg.inv(rotation_matrix), rotation_matrix.T):
-        raise ValueError(f'{name} is not valid. Its inverse must equal its transpose.')
+    norm_diff = np.linalg.norm(rotation_matrix @ rotation_matrix.T - np.eye(3), ord='fro')
```

Comment:
Need a comment here, something like # a matrix M is orthogonal if the product of M with its transpose is the identity, i.e. M dot M.T == I